README.md

```diff
@@ -54,6 +54,11 @@ Community discussions happen primarily on the [dev mailing list][dev-list] or on
 
 Iceberg is built using Gradle 5.2.1.
 
+* To invoke a build and run tests: `./gradlew build`
+* To locally check sanity: `./gradlew check`
```

Comment:
Maybe "run checkstyle"?

Reply:
What do you mean? I see CI is using `./gradlew check`.

Reply:
"check sanity" isn't a very clear description of why someone would use this task. Maybe it would be better to keep it simple and have just `./gradlew build` and `./gradlew build -x test`.

Reply:
Aite